# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xvidcap/xvidcap-1.1.7-xextproto.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Description: Fix to build against xextproto >= 7.1.1

--- xvidcap-1.1.7/src/capture.c.orig	2010-10-06 18:11:58.368001191 +0200
+++ xvidcap-1.1.7/src/capture.c	2010-10-06 18:12:22.208001191 +0200
@@ -65,7 +65,7 @@
 #include <sys/ipc.h>
 #include <sys/shm.h>
 #include <X11/extensions/XShm.h>
-#include <X11/extensions/shmstr.h>
+#include <X11/extensions/shmproto.h>
 #ifndef SOLARIS
 #include <X11/extensions/extutil.h>
 #endif     // SOLARIS
